src/shared/i18n/locales/en.json

```
@@ -358,13 +358,28 @@
-        "consumable": "Consumable"
+        "consumable": "Consumable",
+        "material": "Material"
       }
     },
+    "items": {
+      "material-leather-scrap": "Leather Scrap",
+      "material-cloth-scrap": "Cloth Scrap",
+      "material-metal-scrap": "Metal Scrap",
+      "material-mithril-dust": "Mithril Dust"
+    },
+    "errors": {
```

Biome (2.3.13)

[error] The key errors was already declared.

This is where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)